axelrod/result_set.py

```diff
@@ -722,7 +751,8 @@ def summarise(self):
 
         summary_data = []
         for rank, i in enumerate(self.ranking):
-            data = list(summary_measures[i]) + state_prob[i] + state_to_C_prob[i]
+            data = list(summary_measures[i]) + state_prob[i] + state_to_C_prob[
+                i]
```

Comment:
Can you run black on the final commit? Some of the line wraps seem odd.

Reply:
Yep.  I'm using a new IDE, but just added black, so it should be better going forward.